docs/en/operations/table_engines/mergetree.md

```diff
@@ -430,6 +434,18 @@ ALTER TABLE example_table
 
 When data in a table expires, ClickHouse deletes all corresponding rows.
```

Comment:
this sentence is kind of incorrect now and probably could be just removed

Reply:
That's right.